Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -482,6 +489,10 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 NodejsPackage.Instance.CheckSurveyNews(false);
                 ModulesNode.ReloadHierarchySafe();
 
+#if DEV14
+                TryToAcquireTypings(new[] { "node" });
```

Comment:
Yes, when I was debugging I see `Reload` is always call before the npm stabilization callback. We can't easily avoid having two tsd calls in that case, one on `Reload` to get `node.d.ts` and one after we pick up all the node modules to install their `typings`.

Is that what you were asking?